specs/MultiProfile.md

```diff
@@ -29,6 +29,21 @@ Users can use this interface to get the cookie manager, which is shared by all W
 with this profile. The cookie manager got from profile (CoreWebView2.Profile.CookieManager) is the
 same as that got from CoreWebView2 (CoreWebView2.CookieManager).
 
+Currently, we already have **CoreWebView2Settings** interface to manage password-autosave and
+general-autofill, but it will take effect not immediately but after the next navigation, and it can only
+apply for current WebView2, which means if we start a new WebView2 using the same profile, all the
+settings are default values and cannot be set from the profile. By adding password-autosave and
+general-autofill management interfaces in profile, we can manage the properties and they will apply
+immediately if we set a new value, and all WebView2s that created with the same profile can share
+the settings, which means if we change password-autosave or general-autofill property in one WebView2,
+the others with the same profile will also take effect. And these two property is linked with the same
+properties in ICoreWebView2Settings, so changing one will change the other. Explain it in detail,
+it will take effect immediately no matter setting the properties in **CoreWebView2Settings** or
+**CoreWebView2Profile**, and when the property is changed in one interface, the same property in
+the other interface is changed as well immediately. So for the WebView2s with the same profile,
+their **IsPasswordAutosaveEnabled** or **IsGeneralAutofillEnabled** property
+in **CoreWebView2Settings** and **CoreWebView2Profile** should always keep in sync.
```

Comment:
Is this a two-way binding?  From the beginning part of this paragraph I thought we're adding a way to setup different "initial values" at the Profile level.  Is our customer ask "let me control the default values" or is it "let me affect all webview instances by changing the settings on one instance"?

Reply:
Yes, it's a two-way binding. I'm not very sure, but I personally suppose the customer wants a way to affect all WebView instances by changing the profile instance.

Reply:
This is more about us realizing the existing properties on the CoreWebView2Settings aren't behaving like CoreWebView2Settings but more like properties on CoreWebView2Profile and so 'moving' them (not really moving because of course we can't remove the CoreWebView2Settings properties) to the CoreWebView2Profile.